src/data/items.js

```diff
@@ -325,6 +326,31 @@ export const tomato = crop({
   name: 'Tomato',
 })
 
+/**
+ * @property farmhand.module:items.jalapenoSeed
+ * @type {farmhand.item}
+ */
+export const jalapenoSeed = crop({
+  cropType: JALAPENO,
+  cropTimetable: {
+    [SEED]: 2,
+    [GROWING]: 3,
+  },
+  growsInto: 'jalapeno',
+  id: 'jalapeno-seed',
+  name: 'Jalapeño Seed',
+  tier: 4,
+})
+
+/**
+ * @property farmhand.module:items.carrot
```

Comment:
This needs to be changed to `@property farmhand.module:items.jalapeno`.

Reply:
oops, caught me doing some copypasta 😅 . thanks for catching that